requirements.txt

```
@@ -9,7 +9,7 @@ libsass==0.17.0
 Genshi==0.7.7  # still required by some plugins
 
 # Trac plugins
-TracSpamFilter==1.2.7
+svn+https://svn.edgewall.org/repos/trac/plugins/1.4/spam-filter
```

You can install the zip directly, that way you don't need subversion as a requirement in the Dockerfile
```suggestion
https://trac.edgewall.org/browser/plugins/1.4/spam-filter?format=zip
```